mp4.go

```diff
@@ -201,7 +640,7 @@ func (m metadataMP4) readAtomData(r io.ReadSeeker, name string, size uint32, pro
 		if len(b) < 1 {
```

Comment:
```suggestion
		if len(b) == 0 {
```

Reply:
Saw this whilst reading through, makes more sense to be `== 0`, especially since it's done again below :-)

Reply:
agree